app/assets/stylesheets/config/_colors.scss

```diff
@@ -4,11 +4,18 @@
 
 /* Monochrome
 ------------------------------------------*/
-$white:         rgb(255, 255, 255);
+$white:         rgb(255, 255, 255); // #fff, #ffffff
 $light-grey:    rgb(241, 241, 241);
 $dolphin-grey:  rgb(149, 165, 166);
 $black:         rgb(50,   50,  50);
 $true-black:    rgb(0,     0,   0);
+$slate-grey:    rgb(69, 85, 100); // #455564
+$medium-grey:   rgb(140, 140, 140); // #8c8c8c
+// #7f7f7f
+// #d8d8d8
+// #eff1f4
```

Comment:
should you remove these commented out hex colors?

Reply:
I thought I would leave them because we should create variables for the other hex colors (if we are going to follow the linter suggestions)

Reply:
Alright sounds good.